nio_cli/commands/buildspec.py

```diff
@@ -71,7 +74,9 @@ def _build_properties_spec(self, block):
             property_spec = {}
             if property["default"]:
                 property_spec["default"] = property["default"]
-            properties_spec[property["title"]] = property_spec
+            properties_spec[k] = property_spec
+            properties_spec[k]["title"] = properties[k]['title']
```

Comment:
This would make more sense to be a couple lines up and be `property_spec["title"] = property["title"]`